com.unity.cinemachine/Tests/Runtime/Cinemachine.Runtime.Tests.asmdef

```diff
@@ -15,7 +15,8 @@
     "allowUnsafeCode": false,
     "overrideReferences": true,
     "precompiledReferences": [
-        "nunit.framework.dll"
+        "nunit.framework.dll",
+        "cinemachine_clipper_library.dll"
```

Comment:
Confiner2DUnitTests depend on clipper, and they failed after not including it in some editor versions.